internal/metrics/metrics.go

```diff
@@ -58,11 +64,17 @@ type Meta struct {
 const (
 	BuildInfoGauge = "contour_build_info"
 
-	HTTPProxyTotalGauge     = "contour_httpproxy_total"
-	HTTPProxyRootTotalGauge = "contour_httpproxy_root_total"
-	HTTPProxyInvalidGauge   = "contour_httpproxy_invalid_total"
-	HTTPProxyValidGauge     = "contour_httpproxy_valid_total"
-	HTTPProxyOrphanedGauge  = "contour_httpproxy_orphaned_total"
+	DeprecatedHTTPProxyTotalGauge     = "contour_httpproxy_total"
+	DeprecatedHTTPProxyRootTotalGauge = "contour_httpproxy_root_total"
+	DeprecatedHTTPProxyInvalidGauge   = "contour_httpproxy_invalid_total"
+	DeprecatedHTTPProxyValidGauge     = "contour_httpproxy_valid_total"
+	DeprecatedHTTPProxyOrphanedGauge  = "contour_httpproxy_orphaned_total"
+
+	HTTPProxyTotalGauge     = "contour_httpproxy"
+	HTTPProxyRootTotalGauge = "contour_httpproxy_root"
+	HTTPProxyInvalidGauge   = "contour_httpproxy_invalid"
+	HTTPProxyValidGauge     = "contour_httpproxy_valid"
+	HTTPProxyOrphanedGauge  = "contour_httpproxy_orphaned"
```

Comment:
We should update the [example grafana dashboard](https://github.com/projectcontour/contour/blob/main/examples/grafana/02-grafana-configmap.yaml) to use the new metric names.

Reply:
Thanks. Fixed